src/site/content/en/blog/embed-best-practices/index.md

```diff
@@ -0,0 +1,314 @@
+
+    The [Static map maker](https://staticmapmaker.com/google/) tool helps to configure the parameters required for the URL and gives you the code for the image element in real-time.
+
+    The following snippet shows code for an image with the source set to a Maps Static API URL. It has been included in a link tag that ensures that the actual map can be accessed by clicking on the image. (Note: API key attribute is not included in the url)
+
+    ```html
+    <a href="https://www.google.com/maps/place/Albany,+NY/">
+    <img src="https://maps.googleapis.com/maps/api/staticmap?center=Albany,+NY&zoom=13&scale=1&size=600x300&maptype=roadmap&format=png&visual_refresh=true" alt="Google Map of Albany, NY">
+    </a>
+    ```
+
++ Twitter screenshots
+    Similar to map screenshots, this concept allows you to dynamically embed a Twitter screenshot instead of the live feed. [Tweetpik](https://tweetpik.com/) is one of the tools that can be used to take screenshots of tweets. Tweetpik API accepts the URL of the tweet and returns an image with its contents. The API also accepts parameters to customize the background, colors, borders, and dimensions of the image.
+
+#### Use click-to-load to enhance facades
+
+The click-to-load concept combines lazy-loading and facades. The page initially loads with the facade. When the user interacts with the static placeholder by clicking on it, the third-party embed is loaded. This is also known as the [Import on Interaction](https://addyosmani.com/blog/import-on-interaction/) pattern and can be implemented using the following steps.
+
+1. On page load: facade or static element is included on the page.
+2. On mouseover: facade preconnects to the third-party embed provider.
+3. On Click: the facade is replaced by the third-party product.
+
+Facades may be used with third-party embeds for video players, chat widgets, authentication services, and social media widgets. YouTube video embeds that are just images with a play button are facades that we come across frequently. The actual video loads only when you click on the image. 
+
+You can build a custom click-to-load facade using the _Import on Interaction_ pattern or use one of the following open source facades available for different types of embeds.
+
++ YouTube facade
+
+    [Lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed) is a recommended facade for the YouTube player, which looks like the real player but is 224 times faster. It can be used by downloading the script and stylesheet and then using the `<lite-youtube>` tag in HTML or JavaScript. Custom player parameters supported by YouTube may be included through the `params` attribute.
+
+    ```html
+    <lite-youtube videoid="ogfYd705cRs" playlabel="Play: Keynote (Google I/O '18)"></lite-youtube>
+    ```
+	Following is a comparison between the lite-youtube-embed and the actual embed.
+
+	<div class="w-table-wrapper">
```

Comment:
Replace the table with the column component here as well https://web.dev/handbook/web-dev-components/#columns